src/components/EditCoursePage/CourseSkills.jsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Badge } from '@edx/paragon';
+
+class CourseSkills extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      skillNames: props.input.value || [],
+    };
+  }
+
+  render() {
+    const {
+      className,
+      id,
+      input: {
+        name,
+      },
+      label,
+    } = this.props;
+    const {
+      skillNames,
+    } = this.state;
+    return (
+      <div className="form-group mb-2">
+        <div id={id} name={name} tabIndex="-1">{label}</div>
+        {skillNames.map(skill => (
+          <Badge
+            key={skill.id}
```

Comment:
`skillNames` is a list of strings so `skill.id` will return null. If skill is guaranteed to be unique which I think is, you can use `key={skill}` here.

Reply:
Done